docs/usage/server.rst

```diff
@@ -44,25 +73,9 @@ After that, these management commands are available in the Skill console.
             ; same as above, the order does not matter
             pyStartServer ?logLevel "INFO" ?id "foo"
 
-    .. warning::
-
-        The log levels ``"DEBUG"`` and ``"INFO"`` are not recommended, because then
-        the time for a round-trip between the client and the server is effectively
-        two to three times as long!
-
-    The ``singleMode`` parameter allows you to disable simultaneous connections to
-    the server. By default, multiple connections are allowed for convenience reasons.
-    Especially jupyter users will benefit from this, since jupyter keeps the socket
-    connections open.
-
-    .. warning::
-
-        Even when ``singleMode`` is disabled it is **never** safe to simultaneously
-        access the server. This will lead to strange errors, where variables don't
-        contain what you initially assigned to them.
-
-        In order to stay safe: **never** interleave commands from two different
-        connections.
+            ; this tells the server to wait at most 10.5 seconds
```

Comment:
Why 10.5?

Reply:
To show that a float is possible.

Reply:
Then the code snippet should pass 10.5 too.